### ISSUE TITLE
Vweb Static files are not well served anymore

### ISSUE THREAD

**Issue by Lathanao** [vlang/v#14899]
**V doctor:**
```
OS: linux, Ubuntu 22.04 LTS
Processor: 12 cpus, 64bit, little endian, Intel(R) Core(TM) i7-8750H CPU @ 2.20GHz
CC version: cc (Ubuntu 11.2.0-19ubuntu1) 11.2.0

vexe mtime: 2022-06-30 14:01:19
V full version: V 0.3.0 46f94e8.74bb5ae

Git version: git version 2.34.1
Git vroot status: 0.3-2-g74bb5ae1-dirty
.git/config present: true
thirdparty/tcc status: thirdparty-linux-amd64 eee03626
```

**What did you do?**
```v
module main

import os
import vweb

const (
	port = 8882
)

struct App {
	vweb.Context
}

fn main() {

	mut app := App{}

	if !os.is_dir(os.resource_abs_path('static')) {
		os.mkdir(os.resource_abs_path('static'))?
		os.mkdir(os.resource_abs_path('static/css'))?
		os.create(os.resource_abs_path('static/hello.css'))?
		os.create(os.resource_abs_path('static/css/style.css'))?
	}

	app.mount_static_folder_at(os.resource_abs_path('static'), '/static')
	vweb.run<App>(&app, port)
}

pub fn (mut app App) index() vweb.Result {
	return app.text('yo')
}

pub fn (mut app App) before_request() {
	println(app.static_files)
}
```

**What did you expect to see?**
Static files well served.
and debug show:
```
[Vweb] Running app on http://localhost:8882/
{
'/static/css/style.css': '/var/www/vproject/lew.1/static/css/style.css',
'/static/hello.css': '/var/www/vproject/lew.1/static/hello.css'
}
```

**What did you see instead?**
Static files are not well served and debug show:
```
[Vweb] Running app on http://localhost:8882/
{
'/staticcss/style.css': '/var/www/vproject/lew.1/static/css/style.css',
'/static/hello.css': '/var/www/vproject/lew.1/static/hello.css'
}
```

A '/' is missing in the path of my static file "style.css".
This bug was introduce 9 days ago by #14806.

Just test to revert #14806, and static files are well working again.

**Comment by Lathanao** [comment]
Maybe could check more deeply https://github.com/vlang/v/pull/13361/files

It seen than:
Before #13361:
- '/' in path was every time remove by [1..] line 647
After #13361:
- '/' in path was every time added by '/$trim_mount_path' line 649

I'm not in this case, and I'm not sure 100% but maybe #14806 try to fix a bug introduce by #13361.

Just for info, I think #13361 is a good fix. Before, I removed manually the "[1..]" line 647 to get something well working. #13361 fix that.

**Comment by medvednikov** [comment]
Fixed.
